sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -359,7 +357,7 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
     // If sidechain has ceased set the flag
     hasCeasedOpt.foreach(_ => updateList.add(new JPair(ceasingStateKey, new ByteArrayWrapper(Array.emptyByteArray))))
 
-    scTxCommitmentTreeRootHash.foreach(hash => updateList.add(new JPair(getSidechainTxCommitmentTreeHashKey(hash), hash)))
+    scTxCommitmentTreeRootHashesSeq.foreach(hash => updateList.add(new JPair(getSidechainTxCommitmentTreeHashKey(hash), hash)))
```

Comment:
check the method getSidechainTxCommitmentTreeHashKey(...)
must generate a different key for every hash